### PR TITLE
add Cody Pro SLA

### PR DESCRIPTION
**Pull request by enriquegh**
Adds new entry to our SLAs for support for Cody Pro users

### CODE REVIEW

File: docs/sla/index.mdx [hunk 1]
```diff
@@ -30,6 +30,17 @@ The following policy applies to both our cloud-based (managed instance) and on-p
 
 >NOTE: Premium support with enhanced SLAs can be added to your Enterprise plans as an add-on. Our business hours, defined as Sunday 2 PM PST to Friday 5 PM PST, align with our 24x5 support coverage.
 
+### For Cody Pro plans
+
+| Severity level |                                                                     Description                                                                      |                    Response time                    | Support availability |
```
Comment (MaedahBatool) [inline docs/sla/index.mdx:35]:
```suggestion
| **Severity level** |                                                                     **Description**                                                                      |                    **Response time**                    | **Support availability** |
```

File: docs/sla/index.mdx [hunk 2]
```diff
@@ -30,6 +30,17 @@ The following policy applies to both our cloud-based (managed instance) and on-p
...
+| -------------- | ---------------------------------------------------------------------------------------------------------------------------------------------------- | --------------------------------------------------- | -------------------- |
+| 0              | Emergency: Total loss of service or security-related issue                                                                           | Within one business day of identifying the issue | 8am to 5pm PST (Monday-Friday) |
+| 1              | Severe impact: Billing issues, login issues, refunds | Within one business day of identifying the issue  |8am to 5pm PST (Monday-Friday) |
+| 2              | Medium impact: Core features are unavailable or somewhat slowed; workaround exists                                                                   | Within two business days of identifying the issue   | 8am to 5pm PST (Monday-Friday) |
+| 3              | Minimal impact: Questions or clarifications regarding features, documentation, or deployments                                                        | Within three business days of identifying the issue | 8am to 5pm PST (Monday-Friday) |
```
Comment (MaedahBatool) [inline docs/sla/index.mdx:40]:
```suggestion
| 3              | Minimal impact: Questions or clarifications regarding features, documentation, or deployments                                                        | Within three business days of identifying the issue | 8 AM to 5 PM PST (Monday-Friday) |
```

File: docs/sla/index.mdx [hunk 3]
```diff
@@ -30,6 +30,17 @@ The following policy applies to both our cloud-based (managed instance) and on-p
 
 >NOTE: Premium support with enhanced SLAs can be added to your Enterprise plans as an add-on. Our business hours, defined as Sunday 2 PM PST to Friday 5 PM PST, align with our 24x5 support coverage.
 
+### For Cody Pro plans
+
+| Severity level |                                                                     Description                                                                      |                    Response time                    | Support availability |
+| -------------- | ---------------------------------------------------------------------------------------------------------------------------------------------------- | --------------------------------------------------- | -------------------- |
+| 0              | Emergency: Total loss of service or security-related issue                                                                           | Within one business day of identifying the issue | 8am to 5pm PST (Monday-Friday) |
```
Comment (MaedahBatool) [inline docs/sla/index.mdx:37]:
```suggestion
| 0              | Emergency: Total loss of service or security-related issue                                                                           | Within one business day of identifying the issue | 8 AM to 5 PM PST (Monday-Friday) |
```

Comment (MaedahBatool) [inline docs/sla/index.mdx:37]:
```suggestion
| 0              | Emergency: Total loss of service or security-related issue                                                                           | Within one business day of identifying the issue | 8 AM to 5 PM PST (Monday-Friday) |
```

File: docs/sla/index.mdx [hunk 4]
```diff
@@ -30,6 +30,17 @@ The following policy applies to both our cloud-based (managed instance) and on-p
...
+| 1              | Severe impact: Billing issues, login issues, refunds | Within one business day of identifying the issue  |8am to 5pm PST (Monday-Friday) |
```
Comment (MaedahBatool) [inline docs/sla/index.mdx:38]:
```suggestion
| 1              | Severe impact: Billing issues, login issues, refunds | Within one business day of identifying the issue  |8 AM to 5 PM PST (Monday-Friday) |
```

File: docs/sla/index.mdx [hunk 5]
```diff
@@ -30,6 +30,17 @@ The following policy applies to both our cloud-based (managed instance) and on-p
...
+| 2              | Medium impact: Core features are unavailable or somewhat slowed; workaround exists                                                                   | Within two business days of identifying the issue   | 8am to 5pm PST (Monday-Friday) |
```
Comment (MaedahBatool) [inline docs/sla/index.mdx:39]:
```suggestion
| 2              | Medium impact: Core features are unavailable or somewhat slowed; workaround exists                                                                   | Within two business days of identifying the issue   | 8 AM to 5 PM PST (Monday-Friday) |
```

File: docs/sla/index.mdx [hunk 6]
```diff
@@ -30,6 +30,17 @@ The following policy applies to both our cloud-based (managed instance) and on-p
...
+| 3              | Minimal impact: Questions or clarifications regarding features, documentation, or deployments                                                        | Within three business days of identifying the issue | 8am to 5pm PST (Monday-Friday) |
+
+>NOTE: Cody Free users are not eligible for email Support but can get assistance through our Community Discord.
```
Comment (MaedahBatool) [inline docs/sla/index.mdx:42]:
```suggestion
<Callout type="note">Cody Free users are not eligible for email Support but can get assistance through our Community Discord.</Callout>
```

**Review by MaedahBatool — APPROVED** [review]
Very small formatting fixes, rest look good. Approving the PR to unblock you. :)

**Comment by chenkc805** [comment]
Thanks for putting this together @enriquegh . To make sure I understand, these guarantees are purely around response times, not actual bug fix times or service uptimes, is that right? 

If it's the former, this should be fine. But if it's the latter, then we should loop in @unknwon and @vdavid to get their sign offs.

Unrelated questions:

- What's a "Business" plan here? https://github.com/sourcegraph/docs/blob/ef2d9cb9d70012ca3c65db86a637a950dc9d2848/docs/sla/index.mdx#L46
- What's an "Enterprise plus" or "Elite" plan here? https://github.com/sourcegraph/docs/blob/ef2d9cb9d70012ca3c65db86a637a950dc9d2848/docs/sla/index.mdx#L93

File: docs/sla/index.mdx (hunk 4 above)
Comment (chenkc805) [reply docs/sla/index.mdx:38]: Let's remove "refunds" from this, we don't offer refunds. We can offer free months of Cody Pro but I don't think we should publicly make that known.

**Review by chenkc805 — CHANGES_REQUESTED** [review]

**Comment by chenkc805** [comment]
Hey @enriquegh , just double checking on my earlier question:
> To make sure I understand, these guarantees are purely around response times, not actual bug fix times or service uptimes, is that right?

**Comment by enriquegh** [comment]
> Hey @enriquegh , just double checking on my earlier question:
> 
> > To make sure I understand, these guarantees are purely around response times, not actual bug fix times or service uptimes, is that right?

Sorry @chenkc805 missed that question. That's correct. This about first response time exclusively and we cannot guarantee on resolution time or anything else.

**Review by chenkc805 — APPROVED** [review]

**Comment by MaedahBatool** [comment]
@enriquegh I see some merge conflicts. If the PR is final can you please merge it otherwise it might become stale with the docs release scheduled for Feb 15th. 🙏

**Comment by vdavid** [comment]
Thanks, it's a great start, and we'll iterate from here.

**Pull request merged**